Crons initial docs (#5985)
getsentry/sentry-docs: src/docs/product/crons/alerts.mdx

---
title: Alerts
sidebar_order: 2
---

## Overview

> When your recurring job fails to check-in on schedule or runs beyond its configured max runtime Sentry will create an error event with a `monitor.id` tag. To alert on these error events you can utilize the existing issue alerts feature in Sentry. Here is a brief guide on how to set it up specifically for cron monitors.

Reviewer: ```suggestion
When your recurring job fails to check-in on schedule or runs beyond its configured max runtime, Sentry will create an error event with a `monitor.id` tag. To alert on these error events, you can utilize the existing issue alerts feature in Sentry. Here's a brief guide on how to set it up for Cron Monitors.
```

> ## Step 1: Set filters

Reviewer: ```suggestion
## Step 1: Set Filters
```

> Navigate to the alerts tab in Sentry and create a new issue alert. You should then be presented with the issue alert builder. From here, you can start configuring your cron monitor alert rule. First select the project corresponding to your cron monitor. Then in the `IF` section of the alert rule configuration, add a new filter. You will want to select the filter labeled "The event's tags match {key} {match} {value}".

Reviewer: ```suggestion
Navigate to the alerts tab in Sentry and create a new issue alert. You should see the issue alert builder. From here, you can start configuring your cron monitor alert rule. First, select the project corresponding to your cron monitor. Then, in the `IF` section of the alert rule configuration, add a new filter. You'll want to select the filter labeled "The event's tags match {key} {match} {value}".
```

![Cron alert filter](cron-alert-filter.png)

> Fill in the `key` text field with `monitor.id`, then set that to `equal` the id of the cron monitor that you would like to be alerted on. You can find your cron monitor's id in the cron monitor's detail page under the name.

Reviewer: ```suggestion
Fill in the `key` text field with `monitor.id`, then have it `equal` the id of the cron monitor you'd like to be alerted on. You'll find your cron monitor id in the cron monitor detail page.
```

![Example cron id](cron-id-example.png)

After filling in those fields your finished alert rule filter should look something like:

![Cron completed alert filter](cron-completed-alert-filter.png)

> ## Step 2: Choose notification actions

Reviewer: ```suggestion
## Step 2: Choose Notification Actions
```

> From here you can finalize your alert rule by adding actions and assigning the alert rule to a team. More information on customizing alert rule actions can be found [here](/product/alerts/create-alerts/issue-alert-config/#then-conditions-actions).

Reviewer: ```suggestion
Finalize your alert rule by adding actions and assigning the alert rule to a team. More information on customizing alert rule actions can be found [here](/product/alerts/create-alerts/issue-alert-config/#then-conditions-actions).
```

> Finally, save your newly created issue alert rule and you should get alerts on new errors created from your cron monitor!

Reviewer: ```suggestion
Lastly, save your newly created issue alert rule. You should start getting alerts on new errors created from your cron monitor!
```
